site/src/sphinx/server-annotated-service.rst

```diff
@@ -604,6 +604,59 @@ as follows.
         }
     }
 
+Even if you do not specify any :api:`ResponseConverter` annotation, the response object can be converted into
+an :api:`HttpResponse` by one of the following response converters. They convert the response object
```

Comment:
the following response converters which performs the conversion based on the negotiated media type and...?